### ISSUE TITLE
Prompt Issue

### ISSUE THREAD

**Issue by 9Cirno9** [sajjad-sh33/Polyp-SAM-2#1]
Hello, mentor！
Thanks a lot for your great contribution in this work, may I ask you some questions?
1. Is it possible to use the pre-trained weights directly without any training?
2. If so, is the box prompt for each image during testing obtained based on the coordinates of the top-left and bottom-right corners of its corresponding mask?

**Comment by sajjad-sh33** [comment]
Hi, yes, you can use the pre-trained weights directly without any fine-tune training. Actually, the results in the paper are based on the pre-trained weights directly without any additional training.

And yes, we used the ground truth masks to obtain the box coordinates for the testing phase.
Also you can check our new paper that we use YOLO model to extract the box coordinates. (https://github.com/sajjad-sh33/YOLO_SAM2)
